app/views/catarse_bootstrap/projects/index.html.slim

```diff
@@ -0,0 +1,92 @@
+= render 'shared/header_big', overlayer: true
+
+.hero-back.hero-full
+  .w-container
+    .w-row.u-text-center
+      .w-col.w-col-2
+      .w-col.w-col-8
+        .fontweight-semibold.u-marginbottom-20.fontsize-jumbo Tire projetos do papel!
+        .fontsize-large.lineheight-tight.u-marginbottom-30 Ciências, música, quadrinhos, cinema, impacto social e muito mais. Conheça, inspire-se e apoie projetos&nbsp;
+        .w-row
+          .w-col.w-col-3
+          .w-col.w-col-6
+            = link_to 'Apoie um projeto', explore_path(ref: 'home_banner_explore'), class: 'btn btn-large'
+          .w-col.w-col-3
+      .w-col.w-col-2
+
+- if current_user && @projects_near.present?
+  .section.u-marginbottom-40
+    .w-container
+      .w-row.u-marginbottom-20
+        .w-col.w-col-10.w-col-small-6.w-col-tiny-6
+          .fontsize-large.lineheight-looser
+            span.fa.fa-heart.text-success>.
+            | &nbsp;Próximos a você
+        .w-col.w-col-2.w-col-small-6.w-col-tiny-6
+          = link_to 'Ver todos', explore_path(ref: 'home_near', anchor: "near_of/" ) + current_user.address_state, class: 'btn btn-small btn-terciary'
+      .w-row
+        - @projects_near.each do |project|
```

Comment:
Why don't we use the render_projects helper ?
https://github.com/catarse/catarse/blob/bootstrap/app/controllers/application_controller.rb#L29